prototype/storage_api.py

```diff
@@ -0,0 +1,47 @@
+import mmif
+from fastapi import FastAPI
+from enum import Enum
+from pydantic import BaseModel
+from typing import List, Dict
+import os
+import json
+
+app = FastAPI()
+# get post request from user
+# read mmif inside post request, get view metadata
+# store in nested directory relating to view metadata
+
+# TODO: there's probably a better structure for this. Not sure if this works at the moment
+# TODO: but either way should hopefully be changed to utilize mmif operations?
+class Item(BaseModel):
+    views: List[Dict[str, any]]
+    documents: List[Dict[str, any]]
+    data: Dict[str, any]
+
+@app.get("/")
+async def root():
+    return {"message": "Storage api for pipelined mmif files"}
+
+@app.post("/upload_mmif/")
+async def upload_mmif(item:Item):
+    directory = "/mmif_storage/"
```

Comment:
Let's have a separate file to set this directory, it will decouple the logic and configuration and provide easy of configuration 
configuration file can in any of yml, json, ini formats of even be passed as environment variables (and ["dotenv" file](https://fastapi.tiangolo.com/advanced/settings/#reading-a-env-file))